tests/nnstreamer_plugins/unittest_plugins.cc

```diff
@@ -110,6 +110,94 @@
       gst_harness_teardown (h); \
     }
 
+#define GET_MODEL_PATH(model_name) do { \
+  const gchar *root_path = g_getenv ("NNSTREAMER_BUILD_ROOT_PATH"); \
+  \
+  if (root_path == NULL) \
+    root_path = ".."; \
+  \
+  test_model = g_build_filename (root_path, "tests", "test_models", "models", \
+    #model_name, NULL); \
+  ASSERT_TRUE (g_file_test (test_model, G_FILE_TEST_EXISTS)); \
+  } while (0);
+
+/**
+ * @brief Macro for tensor filter auto option test
+ */
+#define TEST_TENSOR_FILTER_AUTO_OPTION_P(pipeline_desc, fw_name) do { \
+  GstElement *gstpipe, *filter; \
+  gchar *prop_string; \
+ \
+  gstpipe = gst_parse_launch (pipeline_desc, NULL); \
+  g_free (str_launch_line); \
```

Comment:
remove this line and add g_free (str_launch_line); for every testcases.
(or g_free (pipeline_desc); in macro)